Symbolica.Extensions.Configuration.FSharp.sln

```diff
@@ -0,0 +1,56 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30114.105
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{7E4F42B4-EC35-4302-B5A3-924891DF4B06}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Symbolica.Extensions.Configuration.FSharp", "src\Symbolica.Extensions.Configuration.FSharp\Symbolica.Extensions.Configuration.FSharp.fsproj", "{F6D355F4-60F2-4267-812A-8D4E8186F826}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{7DF29F10-BCE4-481B-BC20-A1BD9D8B36A9}"

```

Comment:
Assuming all the code is a straight copy the only difference that catches my eye is that this was called tests before.

Reply:
Yeah it was. I was actually going for consistency with the singular nature of `src`, but I see everyone else uses `tests`. I'll change.